app.rb

```diff
@@ -0,0 +1,105 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'person'
+
+class App
+  attr_accessor :books, :student, :teacher, :rental, :person
+
+  def initialize
+    @books = []
+    @student = []
+    @teacher = []
+    @rental = []
+  end
+
+  # list all books
+  def list_books
+    @books.each_with_index { |_books, _i| print " Title: \"#{book.title}\", Author: \"#{book.author}\"\n" }
+  end
+
+  # students and teachers
+  def list_people
+    people = [*@teacher, *@student]
+    people.each_with_index { |person, _i| print " Name: \"#{person.name}\", Age: \"#{person.age}\"\n" }
+  end
+
+  # create person
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2) [Input the number] : '
+    input = gets.chomp.to_i
+    case input
+    when 1
+      print 'Age: '
+      age = gets.chomp.to_i
+      print 'Name: '
+      name = gets.chomp
+      print 'Do you have parental permission? [Y/N]: '
+      parent_permission = gets.chomp
+      case parent_permission.capitalize
+      when 'Y'
+        true
+      else
+        false
+      end
```

Comment:
- [ ]  I think it will be better if you include a function that will handle the wrong input for the parent's permission. When I was trying to add a person, I entered the letter 'z' for parent permission and the person still got created which might cause some errors in the UI later on 👍🏽 .